packages/apps/human-app/frontend/src/pages/worker/jobs/components/my-jobs/mobile/my-jobs-table-mobile.tsx

```diff
@@ -166,7 +168,7 @@ export function MyJobsTableMobile({
                         color={colorPalette.primary.light}
                         variant="body2"
                       >
-                        {getNetworkName()}
+                        {getNetworkName(chainIdsEnabled, d.chain_id)}
```

Comment:
Second argument is not needed anymore

Reply:
@dnechay I think it should be all good now